docs/models_and_schedulers.rst

```diff
@@ -252,3 +252,146 @@ the following happens:
   requests.
 
 .. image:: images/sequence_example2.png
+
+.. _section-ensemble-models:
+
+Ensemble Models
+---------------
+
+With respect to the inference server's schedulers, an *ensemble* model is not
+an actual model. Instead, it specifies the dataflow between models involved as
+:cpp:var:`Step
+<nvidia::inferenceserver::ModelEnsembling::Step>`. The
+scheduler collects the output tensors in each step, provides them as input
+tensors for other steps according to the specification. In spite of that, the
+ensemble model is still viewed as a single model from an external view.
+
+Ensemble models are intended to be used to encapsulate a procedure that involves
```

Comment:
You need a brief description of what an ensemble is at the start of this section. I would move this paragraph to the start and add an initial sentence. "An ensemble model represents a *pipeline* of one or more models and the connection of input and output tensors between those models".